FluentAvalonia/UI/Controls/IconElement/PathIcon.cs

```diff
@@ -9,6 +12,18 @@ namespace FluentAvalonia.UI.Controls
     /// </summary>
     public class PathIcon : IconElement
     {
+        private Matrix _transform = Matrix.Identity;
+        private Geometry _renderedGeometry;
+
+        static PathIcon()
+        {
+            StretchProperty.OverrideDefaultValue<PathIcon>(Stretch.Uniform);
+            StretchDirectionProperty.OverrideDefaultValue<PathIcon>(StretchDirection.DownOnly);
```

Comment:
Make the default value for StretchDirection `Both`, that way default behavior doesn't change. I'm ok with using `Uniform` as the default stretch - that \*shouldn't\* cause any issues I hope (and now behavior can change anyway)

Reply:
Hi @amwx , 

that was one of those things that made me wonder. If I use `Both`, the icon will look like this if you don't specify a width / height (which is the default): 

![image](https://user-images.githubusercontent.com/47110241/166642418-4b9528bc-3b71-4076-97e7-1c181cc27a62.png)

So I made the icon scale down only be default, so it respects the size of the icon itself. The behavior can be controlled via `<Style Selector="ui|PathIcon" />` but not in code I think.  I've added the needed styles, please double-check if you find this solution OK. If not, please guide me. 

Note: In the original code you have: 
[https://github.com/amwx/FluentAvalonia/blob/fccd2716276d5bf5f22fea0dd8e4b695ba3c2c44/FluentAvalonia/UI/Controls/IconElement/PathIcon.cs#L38-L41](https://github.com/amwx/FluentAvalonia/blob/fccd2716276d5bf5f22fea0dd8e4b695ba3c2c44/FluentAvalonia/UI/Controls/IconElement/PathIcon.cs#L38-L41)

Which will measure the path and therefore also only downscale If I am not missing anything.